README.md

```diff
@@ -22,22 +22,44 @@ Interne henvendelser kan sendes via Slack i kanalen #vtp-chatten
 
 # Hvordan komme igang - Oppsett og konfigurasjon
 
-* VTP trenger Keystore og Truststore liggende i mappe .modig på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
+* VTP trenger Keystore og Truststore liggende i mappe `.modig` på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
 
-## Starte server fra IDE
+guide for å sette Keystore og Truststore: https://howtodoinjava.com/maven/how-to-install-maven-on-windows/
```

Comment:
Skal lenke være denne? https://confluence.adeo.no/display/TVF/Sett+opp+keystore+og+truststore+for+lokal+test

Reply:
Bra oppdaget, ja det skulle være den linken ja :)